tests/testthat/test-compiler-ghpages.R

```diff
@@ -45,3 +45,19 @@ test_that("animint2pages raises an error if no GitHub token is present", {
   do.call(Sys.setenv, as.list(env.old))
   file.copy(config.old, config.file, overwrite = TRUE)
 })
+
+test_that("animint2pages() default branch is gh-pages", {
+  whoami <- suppressMessages(gh::gh_whoami())
+  owner <- whoami[["login"]]
+  local_repo_path <- tempfile(pattern = "repo_clone_")
+  gert::git_clone(url = paste0("https://github.com/", owner, "/animint2pages_test_repo.git"), path = local_repo_path)
+  # Check the default branch after clone
+  head_file <- file.path(local_repo_path, ".git", "HEAD")
```

Comment:
instead of looking at the .git/HEAD file, can you please use gert as below? should be less testing code and probably more robust
```r
> gert::git_branch("~/R/animint2")
[1] "chromote-testing"
```

Reply:
Done.